src/scripts/cronjobs/updateBreachesInRemoteSettings.ts

```diff
@@ -99,8 +102,8 @@ if (
 }
 
 (async () => {
-  const allHibpBreaches = (await HIBP.req("/breaches")) as { body: Breach[] };
-  const verifiedSiteBreaches = allHibpBreaches.body.filter((breach) => {
+  const allHibpBreaches = await HIBP.fetchHibpBreaches();
```

Comment:
Note that I replaced the generic `req` (which just sent requests to the given endpoint of the HIBP API) with a specialised `fetchHibpBreaches`, which sends a request to `/breaches` specifically, and thus is able to return a typed response.